### DISCUSSION TITLE
I have a verified school email, but can't reapply for student

### DISCUSSION THREAD

**Discussion by piguyisme** [community/community#39186] [GitHub Education]
I'm trying to reapply for student after I guess it expired. I have a verified school email and I have used it before to sign up for student.

![Screen Shot 2022-11-15 at 12 25 30 PM](https://user-images.githubusercontent.com/20462230/202020251-4658d8f1-22ab-4d01-8cde-ac3753f32472.png)

However, whenever I try to submit the form I get this:

![Screen Shot 2022-11-15 at 12 28 34 PM](https://user-images.githubusercontent.com/20462230/202020339-d4bef06d-a93c-42e1-9c08-40e0db661707.png)

It doesn't ask for additional documentation, but it asks for my location which I grant access to. Not sure why this is happening, I go to a public high school and it fits the requirements for github education. How can I get access to student again?

**Comment by david-engelmann** [comment]
[You can reach out to Github Education support here](https://support.github.com/contact/education)! They can also assist you if you contact them through support@github.com!

**Comment by samus-aran** [comment]
Hi @piguyisme 👋 

Looking at your screenshot is seems that your school is not part of the partner program. We suggest you reach out to your school (if you haven't done so already) and ask them to contact us regarding to their enrolment in the GitHub Campus Program.
